package connectFour.service;

public class CommentException extends RuntimeException{
    public CommentException(Throwable cause) {
        super(cause);
    }
}
